docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -127,9 +125,9 @@ interface EntityDocument {
 
 ### フェデレーションによるプロトコル間アクセス
 
-直接的なプロトコル間アクセスはサポートされていません。プロトコル間でエンティティにアクセスする必要がある場合は、**フェデレーション**(コンテキストソース登録)を使用して、1 つの GeonicDB インスタンスを別のプロトコルのコンテキストプロバイダとして登録してください。詳細については、[フェデレーション](#フェデレーション)セクションを参照してください。
+直接的なプロトコル間アクセスはサポートされていません。プロトコル間でエンティティにアクセスする必要がある場合は、**フェデレーション**(コンテキストソース登録)を使用して、1 つの GeonicDB インスタンスを他のプロトコルのコンテキストプロバイダーとして登録してください。詳細については、[フェデレーション](#federation)セクションを参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)